mappings/net/minecraft/client/util/ScreenshotUtils.mapping

```diff
@@ -1,5 +1,22 @@
 CLASS cvv net/minecraft/client/util/ScreenshotUtils
 	FIELD a LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD b DATE_FORMAT Ljava/text/DateFormat;
+	METHOD a captureFramebuffer (IILctr;)Lcub;
+		ARG 0 width
+		ARG 1 height
+		ARG 2 framebuffer
 	METHOD a getScreenshotFilename (Ljava/io/File;)Ljava/io/File;
 		ARG 0 directory
+	METHOD a captureScreenshot (Ljava/io/File;IILctr;Ljava/util/function/Consumer;)V
```

Comment:
Does this only capture, or also save?

Reply:
`captureAndWriteScreenshot` seems too lengthy

Reply:
saveScreenshot sounds fine to me?